lightly/api/api_workflow_compute_worker.py

```diff
@@ -244,8 +245,22 @@ def create_compute_worker_config(
         request = DockerWorkerConfigV3CreateRequest(
             config=config, creator=self._creator
         )
-        response = self._compute_worker_api.create_docker_worker_config_v3(request)
-        return response.id
+        try:
+            response = self._compute_worker_api.create_docker_worker_config_v3(request)
+            return response.id
+        except ApiException as e:
+            eb = json.loads(e.body)
+            eb_code = eb.get("code")
+            eb_error = eb.get("error")
+            if str(e.status)[0] == "4" and eb_code is not None and eb_error is not None:
+                raise ValueError(
+                    f"Trying to schedule your job resulted in\n"
+                    f">> {eb_code}\n>> {json.dumps(eb_error, indent=4)}\n"
+                    f">> Please fix the issue mentioned above and see our docs "
+                    f"https://docs.lightly.ai/docs/all-configuration-options for more help."
+                ) from None
```

Comment:
```suggestion
                ) from e
```

This will print the full stacktrace with the `ValueError` at the bottom.